src/main/java/com/cedarsoftware/util/io/WriteOptionsBuilder.java

```diff
@@ -0,0 +1,817 @@
+                skipNullFields,
+                allowNanAndInfinity,
+                forceMapOutputAsTwoArrays,
+                enumPublicFieldsOnly,
+                closeStream,
+                showTypeInfo,
+                classLoader,
+                enumWriter,
+                notCustomWrittenClasses,
+                nonRefClasses,
+                filteredMethodNames,
+                aliasTypeNames,
+                customWrittenClasses,
+                excludedFieldNames,
+                includedFieldNames,
+                nonStandardMappings,
+                fieldFilters,
+                methodFilters,
+                accessorFactories);
+    }
+
+    /**
+     * Load custom writer classes based on contents of resources/customWriters.txt.
+     * Verify that classes listed are indeed valid classes loaded in the JVM.
+     *
+     * @return Map<Class < ?>, JsonWriter.JsonClassWriter> containing the resolved Class -> JsonClassWriter instance.
+     */
+    private static Map<Class<?>, JsonWriter.JsonClassWriter> loadWriters() {
+        Map<String, String> map = MetaUtils.loadMapDefinition("customWriters.txt");
+        Map<Class<?>, JsonWriter.JsonClassWriter> writers = new HashMap<>();
+        ClassLoader classLoader = WriteOptions.class.getClassLoader();
+
+        for (Map.Entry<String, String> entry : map.entrySet()) {
+            String className = entry.getKey();
+            String writerClassName = entry.getValue();
+            Class<?> clazz = MetaUtils.classForName(className, classLoader);
+            if (clazz == null) {
+                System.out.println("Class: " + className + " not defined in the JVM, so custom writer: " + writerClassName + ", will not be used.");
+                continue;
+            }
+            Class<JsonWriter.JsonClassWriter> customWriter = (Class<JsonWriter.JsonClassWriter>) MetaUtils.classForName(writerClassName, classLoader);
+            if (customWriter == null) {
+                throw new JsonIoException("Note: class not found (custom JsonClassWriter class): " + writerClassName + ", listed in resources/customWriters.txt as a custom writer for: " + className);
+            }
+            try {
+                JsonWriter.JsonClassWriter writer = (JsonWriter.JsonClassWriter) MetaUtils.newInstance(customWriter, null);
+                writers.put(clazz, writer);
+            } catch (Exception e) {
+                throw new JsonIoException("Note: class failed to instantiate (a custom JsonClassWriter class): " + writerClassName + ", listed in resources/customWriters.txt as a custom writer for: " + className);
+            }
+        }
+        return writers;
+    }
+
+    /**
+     * Load custom writer classes based on contents of resources/customWriters.txt.
+     * Verify that classes listed are indeed valid classes loaded in the JVM.
+     *
+     * @return Map<Class < ?>, JsonWriter.JsonClassWriter> containing the resolved Class -> JsonClassWriter instance.
+     */
```

Comment:
I think the JavaDoc comment above is a copy/paste error.  I think it is supposed to be talking about loading non-Statnadard method Names, but it talks about Custom Writers instead.